### ISSUE TITLE
How to  publish views for editable.

### ISSUE THREAD

**Issue by trancongminh132** [romanzipp/Laravel-Queue-Monitor#78]
Hi  @romanzipp 
Could  you  let  me  know  how  to  publish  views  for custom something?
Thank  you!

**Comment by romanzipp** [comment]
Views can be published by running

```
php artisan vendor:publish --provider="romanzipp\QueueMonitor\Providers\QueueMonitorProvider"
```

**Comment by trancongminh132** [comment]
Hi @romanzipp 
Seem you don't publish views.
I run that command but only migrations and config file published.
https://github.com/romanzipp/Laravel-Queue-Monitor/blob/master/src/Providers/QueueMonitorProvider.php#L32
Please help me confirm again. Thanks!

**Comment by romanzipp** [comment]
Sorry, my bad, you're right! Fixed in [`2.1.5`](https://github.com/romanzipp/Laravel-Queue-Monitor/releases/tag/2.1.5)
